breakout_1.md

```diff
@@ -0,0 +1,182 @@
+GitHub hasn't been updated. This is a cool feature of Git because you can make many changes (commits) without internet
+access to GitHub because you have a local copy of the repository on your computer! Then, when it's convenient, you can 
+send all your commits to the repository on GitHub. This process is called "pushing" commits.
+```
+git push 
+```
+Now if you got to the page for your branch on github, you'll see the changes you made locally! To view your branch:
+
+1. Navigate to the main page of the repository on www.github.com
+
+2. Click the branch selector menu and select your branch name from the dropdown menu:    
+![](media/branch-selection-dropdown.png)
+
+3. Click on the commit 7-digit identifier to view the changes made:    
+![](media/commit-hash.png)
+
+GitHub will show the line-by-line changes made to the script as well as the commit message.
+
+**Before moving on, make sure all team members have fixed their assigned scripts, staged/committed/pushed their changes, 
+and viewed them on GitHub.**
+
+## 3. Merging branches
+At this point, each branch contains their respective fixed subscripts. However, `main_script.m` on GitHub still isn't functional 
+because you haven't merged all these changes together. The way this is accomplished in GitHub is through a process called
+"Pull Requests". Pull requests merge a given branch with the master branch, applying any changes made in the branch. 
+
+##### a. Open Pull Request
+Each member will need to open a pull request for their branch. Navigate to the main repository page on GitHub and select
+the **Pull requests** tab:    
+![](media/PR-tab.png)    
+and select the green "New pull request" button.
+
+Set the "base" branch to `master` and the "compare" branch to your branch from the dropdown menu:    
+![](media/PR-dropdown.png)
+
+GitHub will bring up information about your branch like the number of commits, files changed, and contributors. You will
+also see the line-by-line changes and commit message from earlier. Select the green "Create pull request" button. Then 
+GitHub will ask you to add more information about the branch you're trying to merge. Write an informative title and 
+select the green "Create pull request" button to confirm your decision:    
+![](media/PR-text.png)    
+
+Now, GitHub will compare the changed files in your branch to their original state in the master branch and try to implement
+these changes. In some cases, there may be conflicts where multiple contributors have changed the same line of code. These
+conflicts are typically resolved manually, but good project organization can generally avoid merge conflicts. **If GitHub 
+finds conflicts with your pull request, ask for help from the Tutorial Team before merging the pull request.**
+
+If there are no conflicts (there shouldn't be any), select the green "Merge pull request" button. This will update the master
```

Comment:
Do we want them to review each others stuff?

Reply:
I was thinking about that... It's definitely good practice and worth mentioning. I didn't want to make extra work for the team leader though, so then I thought just doing a self-review would suffice.